vhdbuilder/parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -0,0 +1,367 @@
+#!/bin/bash
```

Comment:
we moved this file to the cse folder, you need to do the same

Reply:
will remove